packages/esm-patient-search-app/src/mpi/utils.ts

```diff
@@ -0,0 +1,45 @@
+import capitalize from 'lodash-es/capitalize';
+
+export function inferModeFromSearchParams(searchParams: URLSearchParams) {
+  return searchParams?.get('mode')?.toLowerCase() == 'external' ? 'external' : 'internal';
+}
+
+export function mapToOpenMRSPatient(fhirPatients: any): any {
```

Comment:
This could also benefit from explicit types on both sides. Presumable `fhirPatients` is a `fhir.Bundle` and each entry is a `fhir.Patient` or similar.

Reply:
I think there's also an OpenMRS patient type in core that can be the return type.